research-software-directory/RSD-as-a-service
# 191 project view

# Project view - Intro section

Closes #191 

Changes proposed in this pull request:

* First part of the project view page with title, subtitle, description and the info panel (aside)
* Compared to legacy page the partners section is moved from the side panel to a separate section "Participating organisations". The approach is identical to software page. 
* Refactoring software and organisation components to be shared with projects
* Upgrading front-end tests to typescript

How to test:

* `docker-compose down --volumes`: remove old db
*  `docker-compose build`: build all images
* `docker-compose up`: start app
* run data migration scripts
* navigate to projects and select one of the projects, you should see a page similar to image bellow
![image](https://user-images.githubusercontent.com/9204081/162143322-077a9513-4169-4b4c-90ab-e0e535a2e84b.png)


PR Checklist:
